src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball with " + color + " " + number;

```

Comment:
![image](https://user-images.githubusercontent.com/117547096/207953413-ae6400a3-1bb5-46e2-8710-c0451897916c.png)

```suggestion
        return "Ball with color: " + color.name() + " and number: " + number;

```